src/managers/RoleManager.js

```diff
@@ -104,6 +105,7 @@ class RoleManager extends CachedManager {
    * @property {PermissionResolvable} [permissions] The permissions for the new role
    * @property {number} [position] The position of the new role
    * @property {boolean} [mentionable] Whether or not the new role should be mentionable
+   * @property {?(BufferResolvable|Base64Resolvable)} [icon] The icon for the role
```

Comment:
Why doesn't this support an EmojiResolvable?

Reply:
I believe `unicode_emoji`(128b9c9) resolves this

Reply:
No it doesn’t, I said guild emojis not unicode emojis. There should be a method that allows you to pass a guild emoji and uploads that as the role icon

Reply:
^ this

Reply:
Added in 4f7c58d